src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -63,20 +61,12 @@ internal partial class Binder
 
             if (conversion.IsIdentity)
             {
-                // identity tuple conversions result in a converted tuple
-                // to indicate that tuple conversions are no longer applicable.
-                // nothing else changes
-                if (source.Kind == BoundKind.TupleLiteral)
-                {
-                    var sourceTuple = (BoundTupleLiteral)source;
+                if (source is BoundTupleLiteral sourceTuple)
```

Comment:
>) [](start = 59, length = 1)

nit: missing braces

Reply:
Not required by coding style guide, in fact it explicitly advocates this style for short ifs like this.  I would shorted the other ifs in this function, but I don't have a reason to touch them.

---
In reply to: [301759611](https://github.com/dotnet/roslyn/pull/37052#discussion_r301759611) [](ancestors = 301759611)